myst_nbs/case_studies/regression_discontinuity.myst.md

```diff
@@ -0,0 +1,221 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.7
+kernelspec:
+  display_name: pymc-dev-py39
+  language: python
+  name: pymc-dev-py39
+---
+
+(regression_discontinuity)=
+# Regression discontinuity design analysis
+
+:::{post} April, 2022
+:tags: regression discontinuity, causal inference, quasi experimental design, counterfactuals
+:category: beginner
+:author: Benjamin T. Vincent
+:::
+
+[Quasi experiments](https://en.wikipedia.org/wiki/Quasi-experiment) involve experimental interventions and quantitative measures. However, unlike regular experimental designs, random assignment of units (e.g. cells, people, companies, schools, states) to test or control groups is not possible for whatever reason. This inability to conduct random assignment poses problems when making causal claims as it makes it harder to argue that any difference between a control and test group are because of an intervention and not because of a confounding factor.
+
+The [regression discontinuity design](https://en.wikipedia.org/wiki/Regression_discontinuity_design) is a particular form of quasi experimental design. It consists of a control and test group, but assignment of units to conditions is chosen based upon a threshold criteria, not randomly. 
+
+:::{figure-md} fig-target
+:class: myclass
+
+<img src="regression_discontinuity.png" alt="regression discontinuity design schematic" class="bg-primary mb-1">
```

Comment:
The problem with the image comes from here. 

```suggestion
:::{figure-md} fig-target

![regression discontinuity design schematic](regression_discontinuity.png)
```

it was adding extra html classes to format the image, among them was `bg-primary` that stands for use the primary color (blue in this theme) as background.

Note: easier to review here, easier to edit in the notebook directly

Note 2: There is no need to use html image here and regular markdown image syntax should be prefered.